index.js

```diff
@@ -1,17 +1,68 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+    
+  }
 
-  add(item) {}
+  add(item) {
+   
 
-  get(pos) {}
+    this.items.push(item);
+    
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
 
-  max() {}
+    this.length = this.items.length;
+     
+  }
 
-  min() {}
+  get(pos) {
+    if (pos < 0 || pos > this.length){
+      throw new Error("_OutOfBounds_");
+    }
+    return this.items[index];
 
-  sum() {}
+  }
 
-  avg() {}
+  max() {
+    if (this.length === 0){
+      throw new Error("EmptySortedList")
+    } else {
+      
+      return Math.max(...this.items)
```

Comment:
```suggestion
      return Math.max(...this.items)
```
Esto te dejo usarlo si me dices que son los tres puntos y ademas porque son necesarios para el Math.max